src/nccl_ofi_rdma.c

```diff
@@ -482,7 +517,24 @@ static inline int get_properties(nccl_net_ofi_device_t *base_dev,
 		_Static_assert(NCCL_OFI_RDMA_COMM_ID_BITS < 31,
 			       "NCCL_OFI_RDMA_COMM_ID_BITS must be less than 31 so max_communicators fits in an integer");
 		props->max_communicators = NCCL_OFI_RDMA_MAX_COMMS;
+	} else {
+		return ret;
+	}
+
+	/* Overwrite default property */
```

Comment:
why do we overwrite the default property?

Reply:
I thought I set it in the common property population function in `net.c` to a default value of `0` and change it in `rdma.c`. Changed it such that SENDRECV and RDMA set it individually--no default anymore.